packages/paymaster/src/interfaces/IPaymaster.ts

```diff
@@ -1,8 +1,9 @@
 import { UserOperation } from "@biconomy/core-types";
-import { PaymasterAndDataResponse } from "../utils/Types";
+import { PaymasterAndDataResponse, UserOpGasResponse } from "../utils/Types";
 
 export interface IPaymaster {
   // Implementing class may add extra parameter (for example paymasterServiceData with it's own type) in below function signature
+  estimateUserOpGas(_userOp: Partial<UserOperation>): Promise<UserOpGasResponse>;
```

Comment:
we shouldn't add this anyway in IPaymaster. Could be in IHybridPaymaster after we identify need

Reply:
Okay removed this, but now when I am calling the this.paymaster I have to set the type to HybridPaymaster in BaseAccount... Which also I feel not correct as sdk will not be generic. Let me know what you think